src/prommis/uky/costing/ree_plant_capcost.py

```diff
@@ -504,66 +499,77 @@ def build_process_costs(
                     mutable=True,
                     initialize=piping_materials_and_labor_percentage,
                     doc="Percentage of BEC used to estimate piping, materials and labor installation costs",
+                    units=pyunits.percent,
```

Comment:
What's the benefit for giving these parameters units of `pyunits.percent` if you're just going to convert them to dimensionless units later on? It may be simpler to just have them be dimensionless from the start. Or maybe I'm overlooking something...

Reply:
Doing this allows the user to pass an integer, e.g. "15" meaning 15%, instead of a decimal. In my opinion, it's more intuitive to think about percentages in that way.

Previously, we explicitly had `15 /100 * pyunits.dimensionless`, and now we have `15 * pyunits.percent` which looks cleaner in the code.